src/Tools/SarifValidator/Options.cs

```diff
@@ -20,5 +20,12 @@ internal class Options
             HelpText = "Path of the SARIF file to validate",
             Required = true)]
         public string InstanceFilePath { get; set; }
+
+        [Option(
+            'l',
+            "log-file-path",
+            HelpText = "Path to SARIF log file containing the results of the validation",
+            Default = "sarif.log")]
```

Comment:
.log is not a sarif extension, .sarif or sarif.json are. you might consider appending a useful suffix to the input file name as a default, MyInput.Sarif -> MyInput.Sarif.ValidationResults.Sarif or something. I know it gets a bit strange due to the input file being sarif as well but I think this looks reasonable. what do you think?